plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoExpressionRewrite.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.mongodb;
+
+import io.trino.plugin.base.expression.ConnectorExpressionRewriter;
+import io.trino.plugin.mongodb.expression.MongoConnectorExpressionRewriterBuilder;
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.expression.Call;
+import io.trino.spi.expression.ConnectorExpression;
+import io.trino.spi.expression.Constant;
+import io.trino.spi.expression.FunctionName;
+import io.trino.spi.expression.Variable;
+import io.trino.spi.type.ArrayType;
+import io.trino.spi.type.BigintType;
+import io.trino.spi.type.BooleanType;
+import org.bson.Document;
+import org.json.JSONException;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.junit.Assert.assertTrue;
+
+public class TestMongoExpressionRewrite
+{
+    private final ConnectorExpressionRewriter<Document> connectorExpressionRewriter;
+
+    public TestMongoExpressionRewrite()
+    {
+        this.connectorExpressionRewriter = MongoConnectorExpressionRewriterBuilder.build();
+    }
+
+    @Test
+    public void testContainsFunctionRewrite()
+            throws JSONException
+    {
+        ConnectorExpression expression = buildContainsExpression("col", 1L);
+
+        Optional<Document> predicate = this.connectorExpressionRewriter.rewrite(
+                Mockito.mock(ConnectorSession.class),
```

Comment:
Instead of using `Mockito` we can use `TestingConnectorSession`

Reply:
done